<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateItemDialog</class>
 <widget class="QDialog" name="UpdateItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Item</string>
  </property>
  <widget class="QLabel" name="labelItemName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name of Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemImage">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirmEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Edit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Quantity</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinQuantity">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>100</y>
     <width>101</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Item Image:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnItemImage">
      <property name="text">
       <string>Load Item Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
